js/models/userMd.js

```diff
@@ -92,6 +93,84 @@ module.exports = Backbone.Model.extend({
       return modGuid;
     });
 
+    // If blocked_guids has no legitmate guids, it returns an array with an empty string.
+    // Let's remedy that.
+    if (response.blocked_guids.length && response.blocked_guids[0] === "") {
```

Comment:
This line fails for on a fresh install, you need to check for response.blocked_guids first, since it doesn't exist until you've saved your settings at least once. 

Or you need to set it to [] if it doesn't exist before this.